blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Dasharo compatible with MSI PRO Z690-A release v1.1.1
+abstract: 'A short overview of changes introduced in v1.1.1
+           release of Dasharo compatible with MSI PRO Z690-A
+	   with a short demo of Qubes OS 4.1.2.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2023-02-24
+archives: "2023"
+
+tags:
+  - firmware
+  - coreboot
+  - Dasharo
+categories:
+  - Firmware
+
+
+---
+
+# Introduction
+
+New year and new exciting news. Dasharo Team has just released a new version of
+Dasharo firmware compatible with MSI PRO Z690-A. Last v1.1.0 version was
+released on 22th of November 2022 and after 3 months the time has come for v1.1.1.
+Let's see what's new.
+
+# Key changes in new release
+
+The past 3 months were mostly spent on debugging and resolving issues reported
+by community on Matrix and
+[GitHub](https://github.com/Dasharo/dasharo-issues/issues/new/choose). By the
+way, if you haven't joined our [Matrix Dasharo space](https://matrix.to/#/#dasharo:matrix.org)
+yet, do it quick! That is where all interesting stuff happens, like
+various technical discussions, hardware setups, firmware configurations and
+encountered issues.
+
+Let's have a look at the changelog.
+
+### Added
+
+- [Early boot DMA protection menu option](https://github.com/Dasharo/dasharo-issues/issues/275)
+
+  Many people reported various issues when DMA protection was introduced in
+  v1.1.0. Apparently certain GPU cards contain broken OptionROMs which cause
+  DMA errors and display initialization problems. Thus the firmware setup will
+  now contain an option to enable/disable the DMA protection on-demand (with
+  default state being disabled to avoid problems with dGPUs). Additionally an
+  option to keep IOMMU enabled during OS control hand-off (actually during the
+  UEFI Exit Boot Services) has been added. This setting may be especially
+  interesting for people seeking for the most secure configuration of the
+  firmware. But keep in mind, your Windows will not boot if you enable this
+  option! Refer to our [firmware setup documentation](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+  for more details.
+
+- ACPI PCI interrupt routing for CPU PCIe Root Ports
```

Comment:
No reference for this and following bug?

Reply:
Nobody reported it and there were no malfunctions because of it. Thus no reference. I found it myself in free time and just filled the missing piece.